src/components/atoms/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import HomeIcon from '@mui/icons-material/Home';
+import Link from '@mui/material/Link';
+import { Meta, StoryFn } from '@storybook/react';
+import { Box, Breadcrumbs, BreadcrumbsProps, Button } from 'src/components';
+
+export default {
```

Comment:
Add `argTypes` to display the correct types of all the props you want to expose from the component.
I believe this way we won't expose unnecessary props such as `itemsAfterCollapse`, `ref`, `slots`, `classes`, etc. And also display the correct UI according to the types of props we want to leave like: `maxItems` and `separator`... I don't see any other prop we would care to display. Any opinions @Alderr? `sx`? I don't think `children` We can't actually do anything by modifying its values in the UI

Reply:
+1